### ISSUE TITLE
Add aria-required and aria-multiselectable props

### ISSUE THREAD

**Issue by ksiler** [microsoft/react-native-windows#12047]
### Summary

On win32, we use the required and multiselectable fields on accessibilityState to indicate when an element is required to be filled out on a form or when user can select more than one element from a group of selectable elements. These props map to the UIA IsRequiredForForm property and UIA CanSelectMultiple property on the Selection pattern, respectively. Since RN is now using aria- props, it would be great to add aria-required and aria-multiselectable to win32 and windows.

### Motivation

Since RN is now using aria- props, it would be great to add aria-required and aria-multiselectable to win32 and windows.

**Comment by chrisglein** [comment]
These aren't on the Fabric list here #11957 because they aren't in core. They should be upstreamed there or be part of the x-plat RFC.
